packages/block-library/src/image/edit.native.js

```diff
@@ -1,11 +1,38 @@
 /**
  * External dependencies
  */
-import { View, Text } from 'react-native';
+import { View, Image } from 'react-native';
+
+/**
+ * Internal dependencies
+ */
+import { MediaPlaceholder } from '@wordpress/editor';
+
+export default function ImageEdit( { attributes } ) {
+	const { url } = attributes;
+
+	const onUploadPress = () => {
+	};
```

Comment:
Minor nit: I'd add some `console.log()` here to indicate this has been pressed, and also we could use a `TODO` comment in code in the function body to mark this as something we need to implement

Reply:
`eslint` seems to not like console.logs:
`error  Unexpected console statement  no-console`
But I'll add the `TODO` comment for future references.

Thank you!

Reply:
Oh! 😅 makes sense 👍 thank you!